### DISCUSSION TITLE
[Feature Request] Allow for default Subject of emails in ngalerts to be changed

### DISCUSSION THREAD

**Discussion by leeft95** [grafana/grafana#37767] [Alerting]
Don't know if this is already possible in the current code base, but would be good to have a way to modify the `defualt.title` template for emails, using a new template definition.

https://github.com/grafana/grafana/blob/87b8a745765ec760ade27012f43595972004ddce/pkg/services/ngalert/notifier/channels/email.go#L67

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
